x-pack/solutions/security/plugins/security_solution/public/asset_inventory/components/onboarding/hover_for_explanation_tooltip.tsx

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import type { EuiToolTipProps } from '@elastic/eui';
+import { EuiToolTip, useEuiTheme } from '@elastic/eui';
+import { css } from '@emotion/react';
+
+/**
+ * A component that adds a tooltip to its children and adds an underline to indicate that
+ * the children can be hovered for more information.
+ */
+export const HoverForExplanationTooltip = ({ children, ...rest }: Partial<EuiToolTipProps>) => {
```

Comment:
leaving a comment in case i'll forget, but i think this should be moved out of the `asset_inventory/components/onboarding` folder and into the general `asset_inventory/components` folder